.env.sample

```diff
@@ -0,0 +1,4 @@
+POSTGRES_HOST=POSTGRES_HOST
+POSTGRES_NAME=POSTGRES_NAME

```

Comment:
Due to official documentation postgres does not work with POSTGRES_NAME it works with POSTGRES_DB